Detectors/TPC/calibration/include/TPCCalibration/CalibdEdx.h

```diff
@@ -130,15 +118,17 @@ class CalibdEdx
   /// Save the histograms to a TTree.
   void writeTTree(std::string_view fileName) const;
 
- private:
-  constexpr static float mipScale = 1.0 / 50.0; ///< Target value for MIP dE/dx
+  constexpr static float mipScale = 1.0 / 50.0; ///< Inverse of target MIP dE/dx value
```

Comment:
```suggestion
  constexpr static float MipScale = 1.0 / 50.0; ///< Inverse of target MIP dE/dx value
```
see https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Variable_Names#Variable_Names